playground/backend/containers/java/Dockerfile

```diff
@@ -15,5 +15,42 @@
 #  See the License for the specific language governing permissions and
 # limitations under the License.
 ###############################################################################
+ARG BASE_IMAGE
+#ARG BEAM_VERSION_ARG=2.33.0
```

Comment:
we can remove comments once we settle on the final version.

Reply:
Removed commented line